build.gradle

```diff
@@ -24,7 +24,7 @@ plugins {
 
 ext {
     dockerImageVersion = release_version
-    cradleApiVersion = '5.0.0-dev-version-5-3911117271-0064b41-SNAPSHOT'
+    cradleApiVersion = '5.0.0-dev-version-5-4252457586-c1a33ab-SNAPSHOT'
```

Comment:
Maybe switch to dev release for dev-version. We can see workflow example in common-j repository

Reply:
@Nikita-Smirnov-Exactpro When can we expect a dev release for cradle library?

Reply:
We can make cradle API dev release after checking backward compatibility with previous release version.